src/atoms/Space/Space.web.js

```diff
@@ -4,11 +4,14 @@ import styled from 'styled-components';
 
 import { makePxValue } from '../../_helpers/theme';
 
-const Space = styled(({ margin, padding, children, ...props }) => {
+const Space = styled(({ margin, padding, children, style, ...props }) => {
   if (React.Children.toArray(children).length !== 1) {
     throw new Error('Expected a single child for Space component');
   }
-  return React.cloneElement(children, props);
+  return React.cloneElement(children, {
+    ...props,
+    style: [style, children.props.style].filter(Boolean),
```

Comment:
React.js does not accept Array as a valid type for ```style``` prop. That is only supported on React Native. I tested this out and it crashes. You can reproduce this by trying to pass inline style to Text in your story.

Fix:

```suggestion
    style: { ...style, ...children.props.style },
```

Reply:
added